### DISCUSSION TITLE
Cannot get Ray to register

### DISCUSSION THREAD

**Discussion by shaqaruden** [permafrost-dev/node-ray#183] [Q&A] [answered]
I am having an issue with getting Ray to register. I followed both sets of standalone browser registration instructions but with either I get 
> Uncaught ReferenceError: Ray is not defined

```html
<script src="https://cdn.jsdelivr.net/npm/node-ray@latest/dist/standalone.min.js"></script>
<script>
    rayInit(window);
    ray('hello world');
</script>
```

**Comment by shaqaruden** [comment]
I also tried downloading a copy of the script and adding in a simple console.log() to the beginning
```js
"use strict"; (() => {
  console.log('tetsing')
  var ls = Object.create; ...
```

and I see that in the console so I know the script is being included and code is executing

**Reply by patinthehat** [reply, marked as answer]
Hi @shaqaruden, thanks for reaching out. Can you try doing this:
```html
<script>
    window.ray = Ray.ray;
    ray('hello world');
</script>
```

And let me know if that works for you.

**Comment by cmoraleda** [comment]
I have the same problem but this solution does not work for me.
